test_bench.py

```diff
@@ -0,0 +1,73 @@
+import os
+
+from stable_baselines3.common.env_checker import check_env
+from stable_baselines3 import PPO
+
+from SelfDriveEnv import Car, Track
+from training_utils import TensorboardCallback, constant_schedule, linear_schedule, run_experiment, testing, with_changes
```

Comment:
Not a big deal but I would make that two lines by using the "\" to indicate a line break.

Reply:
lol yah good point. I'll do it in the next pr. My staging is kinda dirty with some other changes and I'm lazy LOL